node/modules/retrieval.go

```diff
@@ -58,3 +70,74 @@ func HandleRetrievalTransports(lc fx.Lifecycle, l *lp2pimpl.TransportsListener)
 		},
 	})
 }
+
+func NewLoadBalancer(cfg *config.Boost) func(h host.Host) (*loadbalancer.LoadBalancer, error) {
+	return func(h host.Host) (*loadbalancer.LoadBalancer, error) {
+		peerConfig := map[peer.ID][]protocol.ID{}
+		if cfg.Dealmaking.BitswapPeerID != "" {
+			bsPeerID, err := peer.Decode(cfg.Dealmaking.BitswapPeerID)
+			if err != nil {
+				return nil, err
+			}
+			peerConfig[bsPeerID] = []protocol.ID{
+				network.ProtocolBitswap,
+				network.ProtocolBitswapNoVers,
+				network.ProtocolBitswapOneOne,
+				network.ProtocolBitswapOneZero,
+			}
+		}
+		return loadbalancer.NewLoadBalancer(h, peerConfig)
+	}
+}
+
+func HandleLoadBalancer(lc fx.Lifecycle, lb *loadbalancer.LoadBalancer) {
+	lc.Append(fx.Hook{
+		OnStart: func(ctx context.Context) error {
+			log.Debug("starting load balancer")
```

Comment:
I'd suggest we output these logs at the info level. This is an important component.